### ISSUE TITLE
Outlook 2010: Left/Right Quote Characters not Consistently Read when Navigating via Word

### ISSUE THREAD

**Issue by elliott94** [nvaccess/nvda#9905]
Tested with NVDA snapshot alpha-17928,9dfbd2ee with eSpeak and symbol level set to "Some".

To reproduce:

1. Open a new email from within Outlook, and type/paste the following exactly as shown:

“This is a test with NVDA”

2. Jump to the start of the just-entered text, and use Ctrl + left/right arrows to navigate around

You'll notice that whilst NVDA reads the left quote, the right quote is missing - however, both of these characters are read if navigating individually by character.

**Comment by DrSooom** [comment]
Maybe related: Issue #9681

**Comment by Adriani90** [comment]
Fixed by #11167. Testing with NVDA Version: alpha-20400,675c4c54. Closing as works for me.
